### ISSUE TITLE
-webkit-text-fill-color and list marker

### ISSUE THREAD

**Issue by upsuper** [whatwg/compat#71]
See the following code:
```html
<!DOCTYPE html>
<style>
:root { color: red; -webkit-text-fill-color: green; }
li::before, li::after { content: "##"; }
</style>
<ol><li>hello</li></ol>
<ul><li>hello</li></ul>
```

Should the list markers use `color` or `-webkit-text-fill-color`?

Currently all browsers use `color` for the list markers, but it seems inconsistent to me, because in theory, like `::before` and `::after`, list markers are just some pseudo elements contain some generated content. I think it should use the same color as other text.

**Comment by chenpighead** [comment]
IMHO, I prefer that the list markers should use the same color as other text.

IIRC, I implemented -webkit-text-fill-color just for the compatibility reason, so I was targeting to be agree with WebKit and Blink.

@miketaylr could you (or someone) help gather feedback from WebKit and Blink?

**Comment by miketaylr** [comment]
@tabatkins, @grorg any thoughts here?

**Comment by tabatkins** [comment]
Yeah, if we assume text-fill stuff exists, then bullets probably *should* use them for their colors.

**Comment by grorg** [comment]
Agreed.

**Comment by upsuper** [comment]
Filed [bug 1329052](https://bugzilla.mozilla.org/show_bug.cgi?id=1329052) for Gecko.
